README.md

```diff
@@ -369,7 +369,7 @@ When using standard Vue without a build step and mounting to in-DOM templates, i
 - The compiler then compiles the string into a JavaScript render function
 - Vue then replaces existing DOM templates with new DOM generated from the render function.
 
-`petite-vue` avoids all this overhead by walking the existing DOM and attaching fine-grained reactive effects to the elements directly. The DOM _is_ the template. This means `petite-vue` is much more efficient in progressive enhancement scenarios.
+`petite-vue` avoids all this overhead by walking the existing DOM and attaching fine-grained reactive effects to the elements directly. The DOM is the template. This means `petite-vue` is much more efficient in progressive enhancement scenarios.
```

Comment:
Why are you removing the emphasis here?

Reply:
I think it's a bit odd that the `is` isn't consistent with `This means `petite-vue` is much more efficient`.

Reply:
The "is" in the sentence you edited is emphasized to reflect a pattern of speech where a specific word is given, well, extra emphasis when enunciating it. It's common to represent that with italics in writing. It's not a plain "is" like the one from the sentence after it.

As such, I'd recommend reverting this part of the PR to the original markup.

Reply:
Thanks for the explanation, learned :)